tensorflow/compiler/xla/service/platform_util.cc

```diff
@@ -48,9 +51,14 @@ string CanonicalPlatformName(const string& name) {
   if (platform_str == "cpu") {
     platform_str = "host";
   }
-  // "gpu" and "cuda" mean the same thing.
+  // When configured on CUDA, "gpu" and "cuda" mean the same thing.
+  // When configured on ROCm, "gpu" and "rocm" mean the same thing.
   if (platform_str == "gpu") {
+#if GOOGLE_CUDA
```

Comment:
Could we reverse the guard? CUDA stuff for now needs to compile without any special symbols defined.